src/blockstack_cli.rs

```diff
@@ -18,6 +18,7 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
+#![allow(non_fmt_panic)]
```

Comment:
Didn't we fix all these warnings in `master`? If so, we should just merge `master` -> `develop` and leave this warning as is. Technically, this warning is telling us that we have a spurious string construction.

Reply:
FYI: I checked my "fix rust warnings" PR into `develop`, but i still get the warnings in next `branches`.

https://github.com/blockstack/stacks-blockchain/pull/2702

Reply:
Yep -- warnings were still present in tests and other workspace members, PR https://github.com/blockstack/stacks-blockchain/pull/2758 addresses it. We should not disable this warning.

Reply:
Reverted